### PR TITLE
creating of objects

### PR DESCRIPTION
**Pull request by ksvirkou-hubspot**

### CODE REVIEW

File: php/src/Commands/Objects/ObjectsCommand.php [hunk 1]
```diff
@@ -37,4 +37,29 @@ protected function getObjectType(InputInterface $input): string
...
+            )
+        ;
+    }
+
+    protected function getProperties(array $elements): array
+    {
+        $properties = [];
+        foreach ($elements as $element) {
+            $array = explode('=', $element);
+            if (2 != count($array)) {
```
Comment (atanasiuk-hubspot) [inline php/src/Commands/Objects/ObjectsCommand.php]: Please avoid "magic" numbers and create a constant with meaningful name

**Comment by atanasiuk-hubspot** [comment]
Please also provide in description an example of usage of a new command to see how it eventually looks like

**Comment by ksvirkou-hubspot** [comment]
> Please also provide in description an example of usage of a new command to see how it eventually looks like

done

File: README.md [hunk 2]
```diff
@@ -18,14 +18,20 @@ cp node/.env.template node/.env
...
+6. Follow the language instructions on how to run.
 
-```
+```Node
 cd node # there's a README in this folder with instructions
 npm install
 ./bin/cli.js
 ```
 
+```Php
```
Comment (atanasiuk-hubspot) [inline README.md]: I don't think it makes sense to provide setup instructions for all languages in top-level README. lets leave only `node.js` as an example.

**Review by atanasiuk-hubspot — APPROVED** [review]
LGTM

**Pull request merged**